cek/core/models.py

```diff
@@ -231,28 +231,28 @@ def verify_application_id(self, application_id):
 
 
 class LaunchRequest(Request):
-    """Type represents a LaunchRequest from CEK."""
+    """ This request is used to declare that the user has requested to use a this extension. """
     pass
 
 
 class SessionEndedRequest(Request):
-    """Type represents an SessionEndedRequest from CEK."""
+    """ Is send to declare that the user has requested to stop using this extension. """
```

Comment:
How about: `Request received when the user stops the skill or the skill timed-out`?